backends-velox/src/test/scala/org/apache/gluten/execution/FallbackSuite.scala

```diff
@@ -263,4 +264,43 @@ class FallbackSuite extends VeloxWholeStageTransformerSuite with AdaptiveSparkPl
       }
     }
   }
+
+  test("fallback reader with unsupported filesystem") {
```

Comment:
This test case does not seem to see the fallback logic

Reply:
Moved to VeloxScanSuite, thanks.